pynsist: exclude unnecessary DLL files and folders

Mithil467 opened the pull request:
I have added some unnecessary DLL files and folders to the exclude section of the pynsist file and also sorted them.

Related-to: #1094

davvid commented on pynsist.cfg:
```diff
@@ -42,18 +42,70 @@ pypi_wheels = PyQt5==5.14.2
+    pkgs/PyQt5/Qt/bin/Qt5MultimediaWidgets.dll
+    pkgs/PyQt5/Qt/bin/Qt5NetworkAuth.dll
+    pkgs/PyQt5/Qt/bin/Qt5Nfc.dll
+    pkgs/PyQt5/Qt/bin/Qt5OpenGL.dll
```
Thanks for doing this! Please keep Qt5OpenGL in case we end up using bits of it in our QGraphicsScene rendering layer.

davvid commented on pynsist.cfg:
```diff
@@ -42,18 +42,70 @@ pypi_wheels = PyQt5==5.14.2
+    pkgs/PyQt5/Qt/bin/Qt5SerialPort.dll
+    pkgs/PyQt5/Qt/bin/Qt5Sql.dll
+    pkgs/PyQt5/Qt/bin/Qt5Test.dll
+    pkgs/PyQt5/Qt/bin/Qt5WebChannel.dll
```
WebChannel might be something we need to keep. (I bet it's also one of the bigger ones =( )

davvid commented on pynsist.cfg:
```diff
@@ -42,18 +42,70 @@ pypi_wheels = PyQt5==5.14.2
+    pkgs/PyQt5/Qt/bin/libeay32.dll
+    pkgs/PyQt5/Qt/bin/msvcp140.dll
+    pkgs/PyQt5/Qt/bin/opengl32sw.dll
+    pkgs/PyQt5/Qt/bin/ssleay32.dll
```
i'm paranoid so libEGL.dll, libGLESv2.dll, ssleay32.dll and opengl32sw.dll might be worth keeping.

davvid commented:
Done reviewing. Thanks for these updates.

If you could kindly add comments to the file describing how you went about coming up with the exclusion list then that will help us out in the future if we ever have to change it.

Let me know what you think about the sugs. We'l aim to get this merged before the next release.

Mithil467 commented:
I have removed all the DLLs you have mentioned above.  The installer size now is 39.5MB and the pkgs folder is 113 MB. The major ones are opengl32sw.dll and libGLESv2.dll with 20.43 MB and 3.49MB sizes respectively. The rest are pretty small in comparison.

I have also added a few comments explaining the process. Let me know if it is fine, or needs any changes!

davvid commented on pynsist.cfg:
```diff
@@ -42,18 +51,64 @@ pypi_wheels = PyQt5==5.14.2
+    pkgs/PyQt5/QtDesigner.pyd
+    pkgs/PyQt5/QtQml.pyd
+    pkgs/PyQt5/QtQuick.pyd
+    pkgs/PyQt5/QtQuickWidgets.pyd
```
Thanks again and sorry for taking so long to get back to this. I may actually go ahead and remove opengl post-merge (don't worry about it) since that's a hefty one ~ thanks for the note about the sizes. If something breaks we can always add it later.

One last request, though. Please resave this file because and amend the commit after adjusting your editor settings. This file is missing the terminating newline.

https://thoughtbot.com/blog/no-newline-at-end-of-file

Mithil467 replied:
Sure! I have added the terminating newline and also added the `opengl32sw.dll` file in exclusion list.

Mithil467 replied:
Oh wait, the newline is gone after commit. Git is probably removing it, I'll update it.

Mithil467 replied:
Ok I had some confusion but I think now it is done, can you see if the newline is added?

davvid commented:
Thanks!
